test/logflare/backends/clickhouse_adaptor_test.exs

```diff
@@ -82,8 +82,13 @@ defmodule Logflare.Backends.Adaptor.ClickhouseAdaptorTest do
 
       @client
       |> expect(:send, fn req ->
-        send(this, {ref, req[:body]})
-        %Tesla.Env{status: 200, body: ""}
+        expected_url =
+          "http://localhost:8443?query=INSERT%20INTO%20supabase_log_ingress%20FORMAT%20JSONEachRow"
+
+        if req[:url] == expected_url do
+          send(this, {ref, req[:body]})
+          %Tesla.Env{status: 200, body: ""}
+        end
```

Comment:
```suggestion
        assert req[:url] =~ ~r"localhost:8443.+query=INSERT.+EachRow"
        send(this, {ref, req[:body]})
        %Tesla.Env{status: 200, body: ""}
```

Reply:
on a side note, since this only sets additional query params to the end url, we can add a query_params config map as part of the WebhookAdaptor config, which makes it more generalized and removes the need for overriding the entire url